src/libmongoc/src/mongoc/mongoc-database.c

```diff
@@ -853,9 +853,8 @@ mongoc_database_get_collection_names_with_opts (mongoc_database_t *database,
    return ret;
 }
 
-
-mongoc_collection_t *
-mongoc_database_create_collection (mongoc_database_t *database,
+static mongoc_collection_t *
+create_collection (mongoc_database_t *database,
                                    const char *name,
                                    const bson_t *opts,
                                    bson_error_t *error)
```

Comment:
Ensure you're running a formatting pass over the sources before merging, as later unrelated changes to the same file might re-format the previously-unformatted code and glutter the diff. I use VSCode "Format on save" to ensure I never forget and don't need to bother fiddling with whitespace. Although this often causes headaches when an unrelated change in the file will rewrite some other lines that hadn't been given a formatting pass.

We could also take advantage of `.gitblameignore` to do a formatting pass on all-the-things (and also maybe increase our column limit? 👀)